### ISSUE TITLE
<button> with force-expand-multiline formatting bug

### ISSUE THREAD

**Issue by blemaire** [beautify-web/js-beautify#1335]
# Description
> BeautifyJs does not seem to handle <button> elements too well with the `force-expand-multiline` option enabled.

# Input
The code looked like this before beautification:
```
<button
    class="my-class"
    id="id1"
>
    Button 1
</button>

<button
    class="my-class"
    id="id2"
>
    Button 2
</button>
```

# Expected Output
The code should have looked like this after beautification:
```
<button
    class="my-class"
    id="id1"
>
    Button 1
</button>

<button
    class="my-class"
    id="id2"
>
    Button 2
</button>
```

# Actual Output
The  code actually looked like this after beautification:
```
<button
    class="my-class"
    id="id1"
>
    Button 1
    </button>

    <button
        class="my-class"
        id="id2"
    >
        Button 2
        </button>
```

## Environment
OS: MacOS

## Settings
I'm using all the default values apart from the below:

```json
{
    "wrap-attributes": "force-expand-multiline"
}
```

**Comment by blemaire** [comment]
Some changes have occured between the merge of the PR which introduced this feature (#1070) and now meaning this feature is now broken.

The tests for this (at today's date) no longer match the original expected behaviour:
https://github.com/beautify-web/js-beautify/blob/fde05f7e11c3b7ce77a6a8f86a0808d88814a666/js/test/generated/beautify-html-tests.js#L760-L772

The test should be as below:
```js
        opts.wrap_attributes = 'force-expand-multiline';
        opts.wrap_attributes_indent_size = 4;
        opts.wrap_line_length = 80;
        test_fragment('<div  >This is some text</div>', '<div>This is some text</div>');
        test_fragment('<div attr="123"  >This is some text</div>', '<div attr="123">This is some text</div>');
        test_fragment(
            '<div attr0 attr1="123" data-attr2="hello    t here">This is some text</div>',
            //  -- output --
            '<div\n' +
            '    attr0\n' +
            '    attr1="123"\n' +
            '    data-attr2="hello    t here"\n' +
            '>\n' +
            '    This is some text\n' +
            '</div>');
```

**Comment by bitwiseman** [comment]
@blemaire  
I can repro the issue with the input you provided, so this is definitely a bug.  

But I'm not sure I understand your comment about the tests.  Looking at the tests in the original PR they look to me like the first code that you list and not what you think they should look like.

**Comment by blemaire** [comment]
I could be wrong about the tests, the formatting has been changed a lot so somewhat difficult to read. I attempted a fix on this issue but changes are not without many side effects :(

**Comment by bitwiseman** [comment]
@blemaire Yeah, That is a problem with the html beautifier especially right now.

**Comment by bitwiseman** [comment]
Looks like this is fixed in 1.8.0-rc6.
